tests/ImportsSeparatedIntoSideEffectGroups/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -74,6 +76,7 @@ import anotherSameLevelRelativePath1 from "./anotherSameLevelRelativePath1";
 import sameLevelRelativePath1 from "./sameLevelRelativePath1";
 
 import "side-effect-x";
+
```

Comment:
Hmmm, why are these newlines being added?  I tried to get rid of it, and the only way I could was to remove _all_ separators from importOrder.  Is that intentional?  I thought we were only adding blank lines above the side-effect imports when we found a custom separator.

Reply:
These new lines were supposed to be there, according to the comment in tests/ImportsSeparatedIntoSideEffects/ppsi.spec.ts there was a comment:
```
run_spec(__dirname, ['typescript'], {
    importOrder: [
        '^@core/(.*)$',
        '^@server/(.*)',
        '^@ui/(.*)$',
        '^[./]',
        '', // This adds a newline before side effect groups
    ],
});
```
I’m pretty sure this change is just me restoring that expectation!

Reply:
But this is a newline _after_ the side-effect node, not between side-effect groups, right?  That doesn't seem to be what that comment was about.

Reply:
I guess I'm confused now. I'm seeing it as a newline before *any* chunk, in this case the following chunk.

I think it was weird to have a special meaning to the last slot, because side-effect imports aren't "last". We just create a chunk before them even if there are no imports in that chunk, but I guess some people would consider the side-effect chunks the last entry in a chunk?

idk. I'm confused on this now! The sample looks reasonable to me, but I'm not sure how I would configure things around side-effects if I didn't know the implementation.

Reply:
OK, I was thinking about side-effect nodes _starting_ a chunk, but in reality they _become_ a chunk.  Which I guess makes sense.  So, side-effect imports will be broken out on their own as long as there's a custom group separator somewhere in the `importOrder`.  That keeps them from looking like they're part of either the group before or the group after, each of which is sorted according to the `importOrder`.  So, I think the change here is correct after all.  Thanks.